.swiftpm/xcode/xcshareddata/xcschemes/Amplify.xcscheme

```diff
@@ -9,9 +9,9 @@
          <BuildActionEntry
             buildForTesting = "YES"
             buildForRunning = "YES"
-            buildForProfiling = "YES"
-            buildForArchiving = "YES"
-            buildForAnalyzing = "YES">
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
```

Comment:
what are these changes? seems like YES as before may be better

Reply:
oh, these are the Build settings of `Amplify` Target -> `Edit schemes` 
I'll change that